physionet-django/console/views.py

```diff
@@ -934,3 +945,90 @@ def guidelines_review(request):
     Guidelines for reviewers.
     """
     return render(request, 'console/guidelines_review.html')
+
+
+@login_required
+@user_passes_test(is_admin, redirect_field_name='project_home')
+def download_credentialed_users(request):
+    """
+    CSV create and download for database access.
+    """
+    # Create the HttpResponse object with the appropriate CSV header.
+    project_access = DUASignature.objects.filter(project__access_policy = 2)
+    added = []
+    List = [['First name', 'Last name', 'E-mail', 'Institution', 'Country', 
```

Comment:
Not a good name for a variable!

Reply:
I agree.... Fixed in the new PR... among other bugs...